### PR TITLE
 add doc about install slime in hango

### PR DESCRIPTION
**Pull request by MouceL**

### CODE REVIEW

File: modules/bundle_example_hango/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Use distroless as minimal base image to package the manager binary
```
Comment (YonkaFang) [inline modules/bundle_example_hango/Dockerfile:1]: example_hango?

File: modules/bundle_example_hango/install/slime-hango.yaml [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  ports:
+    - name: http
+      port: 80
+      protocol: TCP
+      targetPort: http
+    - name: aux-port
+      port: 8081
+      protocol: TCP
+      targetPort: aux-port
+    - name: log-source-port
```
Comment (YonkaFang) [inline modules/bundle_example_hango/install/slime-hango.yaml:16]: is this port used by hango?

File: modules/bundle_example_hango/install/slimeboot-hango.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+apiVersion: config.netease.com/v1alpha1
+kind: SlimeBoot
```
Comment (YonkaFang) [inline modules/bundle_example_hango/install/slimeboot-hango.yaml:2]: does hango still use slimeboot?

Comment (MouceL) [reply modules/bundle_example_hango/install/slimeboot-hango.yaml:2]: hango use manifest instead of slimeboot

**Review by YonkaFang — APPROVED** [review]

**Pull request merged**